eoscostarica/guide.eoscostarica.io
# Consistent styles

### Check for consistent styles

#131 

### Steps to test
1. Run project locally
1. Go to update files and see content

#### CheckList
